lib/pushmi_pullyu/cli.rb

```diff
@@ -89,6 +95,20 @@ def parse_options(argv)
         config.monitor = true
       end
 
+      opts.on('-rh', '--redis-host IP', 'Host IP of Redis instance to read from.'\
```

Comment:
My only comment might be that the redis options could possibly moved to a separate method, then called here e.g. redis_options(opts)